code/game/gamemodes/miniantags/abduction/machinery/pad.dm

```diff
@@ -0,0 +1,54 @@
+/obj/machinery/abductor/pad
+	name = "Alien Telepad"
+	desc = "Use this to transport to and from human habitat"
```

Comment:
@KasparoVy they're unintegrated ayyliens, they probably view everything as different breeds of humans
that's why they probe them